lib/vrl/compiler/fuzz/fuzz_targets/fuzz_target_1.rs

```diff
@@ -0,0 +1,28 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+use std::collections::BTreeMap;
+use vrl_compiler as vrl;
+
+fuzz_target!(|data: Vec<parser::Program>| {
+    for expr in data {
+        // Compile the VRL.
+        let program = vrl::compile(expr, &vec![]).unwrap();
```

Comment:
Does this take into account that there's probably a lot of ASTs that will not pass static analysis? Or are the programs passed into the closure already pre-validated?

Reply:
That's a good question. Not at the minute. With this an invalid AST would cause a panic which would indicate a test failure. I need to work out how to set it up so invalid ASTs are just ignored..